### ISSUE TITLE
Make conviction accumulate over days

### ISSUE THREAD

**Issue by sembrestels** [1Hive/conviction-voting-app#28]
Right now it's a constant in the smart contract, but we may want to make it configurable, so conviction votings can last hours, days, or weeks.

**Comment by GriffGreen** [comment]
man that gets really confusing... I would stick with one unit... hours, in the backend and do all the math in that one unit always...

But user input should have a major UX focus... I would ask the user an understandable question like:

"How much time does it take to reach 80% of a voter’s max voting power?"

Then have a box where they can put a number... and a dropdown where they can pick minutes, hours, days, weeks, months... 
<img width="237" alt="Screen Shot 2020-04-25 at 5 25 57 PM" src="https://user-images.githubusercontent.com/14945613/80283716-1afe6300-871a-11ea-981f-0cd789fc7044.png">

but for the units of t in all the code... just stick with hours and if someone picks 6 minutes that means 0.1 hours and if someone picks 2 weeks that means 336 hours... and then you do math in the backend to figure out what timestep you actually want and all that.

**Comment by sembrestels** [comment]
Yes, that's the idea. In the frontend we ask for the [halftime](https://en.wikipedia.org/wiki/Half_time_(physics)) (50% seems more appropriate as it's a term commonly used in physics and electronics), and we calculate alpha directly from it with the formula ```α = (1/2)**(1/t)```.

**Comment by sembrestels** [comment]
I have changed default alpha to `0.9999599`, that has a halftime of ~3 days.

I also made the threshold linearly dependent with the max amount of conviction (which changes with alpha). This way, the value of the threshold as a percentage doesn't change depending on alpha.

The previous threshold formula was:
`threshold = (rho * supply) / (beta - (requeted / funds)) ** 2`

And now it is:
`threshold = (rho * supply) / (1 - alpha) / (beta - (requeted / funds)) ** 2`

Any threshold formula available in the future should include this `1/(1-alpha)` component to pair with conviction.
